src/react-components/home-root.js

```diff
@@ -154,6 +154,22 @@ class HomeRoot extends Component {
                   >
                     <FormattedMessage id="home.report_issue" />
                   </a>
+                  <a
+                    className="footer-content__links__link"
+                    target="_blank"
+                    rel="noopener noreferrer"
+                    href="https://github.com/mozilla/hubs/blob/master/TERMS.md"
```

Comment:
Are we actually just going to link to github for the notices? We ought to be able to generate some styled HTML from the markdown at least.

Reply:
Yeah this is what was suggested by legal -- not 100% sure why but seems good to have a ground truth for now